src/turbine/runtime/intermediate.py

```diff
@@ -30,11 +30,11 @@ async def records(self, collection: str, config: dict[str, str] = None) -> None:
             config = {}
 
         if not collection:
-            raise Exception(f"A collection name is required for all resources")
+            raise Exception("A collection name is required for all resources")
```

Comment:
```suggestion
            raise Exception("A collection name is required when calling records()")
```
i feel like this could be a little clearer